exercises/variable-length-quantity/tests/variable-length-quantity.rs

```diff
@@ -0,0 +1,89 @@
+extern crate variable_length_quantity as vlq;
+
+#[test]
+fn to_single_byte() {
+    assert_eq!(&[0x40], vlq::to_bytes(&[0x40]).as_slice());
+    assert_eq!(&[0x7f], vlq::to_bytes(&[0x7f]).as_slice());
+}
+
+#[test]
+#[ignore]
+fn to_single_byte_zero() {
+    assert_eq!(&[0x00], vlq::to_bytes(&[0x00]).as_slice());
+}
+
+#[test]
+#[ignore]
+fn to_double_byte() {
+    assert_eq!(&[0x81, 0x00], vlq::to_bytes(&[0x80]).as_slice());
+    assert_eq!(&[0xc0, 0x00], vlq::to_bytes(&[0x2000]).as_slice());
+    assert_eq!(&[0xa6, 0x37], vlq::to_bytes(&[0x1337]).as_slice());
+    assert_eq!(&[0xff, 0x7f], vlq::to_bytes(&[0x3fff]).as_slice());
+}
+
+#[test]
+#[ignore]
+fn to_tripple_byte() {
+    assert_eq!(&[0x81, 0x80, 0x00], vlq::to_bytes(&[0x4000]).as_slice());
+    assert_eq!(&[0xc0, 0x80, 0x00], vlq::to_bytes(&[0x10_00_00]).as_slice());
+    assert_eq!(&[0xc8, 0xe8, 0x56], vlq::to_bytes(&[0x12_34_56]).as_slice());
+    assert_eq!(&[0xff, 0xff, 0x7f], vlq::to_bytes(&[0x1f_ff_ff]).as_slice());
+}
+
+#[test]
+#[ignore]
+fn to_quadruple_byte() {
+    assert_eq!(&[0x81, 0x80, 0x80, 0x00],
+               vlq::to_bytes(&[0x20_00_00]).as_slice());
+    assert_eq!(&[0xc0, 0x80, 0x80, 0x00],
+               vlq::to_bytes(&[0x08_00_00_00]).as_slice());
+    assert_eq!(&[0xb2, 0xdf, 0xee, 0x4e],
+               vlq::to_bytes(&[0x06_57_f7_4e]).as_slice());
+    assert_eq!(&[0xff, 0xff, 0xff, 0x7f],
+               vlq::to_bytes(&[0x0f_ff_ff_ff]).as_slice());
+}
+
+#[test]
+#[ignore]
+fn to_bytes_multiple_values() {
+    assert_eq!(&[0x40, 0x7f], vlq::to_bytes(&[0x40, 0x7f]).as_slice());
+    assert_eq!(&[0x81, 0x80, 0x00, 0xc8, 0xe8, 0x56],
+               vlq::to_bytes(&[0x4000, 0x12_34_56]).as_slice());
+    assert_eq!(&[0xc0, 0x00, 0xc8, 0xe8, 0x56, 0xff, 0xff, 0xff, 0x7f, 0x00, 0xff, 0x7f, 0x81,
+                 0x80, 0x00],
+               vlq::to_bytes(&[0x2000, 0x12_34_56, 0x0f_ff_ff_ff, 0x00, 0x3fff, 0x4000])
+                   .as_slice());
+}
+
+#[test]
+#[ignore]
+fn multiple_bytes() {
+    assert_eq!(&[0x2000, 0x12_34_56, 0x0f_ff_ff_ff, 0x00, 0x3fff, 0x4000],
+               vlq::from_bytes(&[0xc0, 0x00, 0xc8, 0xe8, 0x56, 0xff, 0xff, 0xff, 0x7f, 0x00,
+                                 0xff, 0x7f, 0x81, 0x80, 0x00])
+                   .as_slice());
+}
+
+#[test]
+#[ignore]
+fn chained_execution_is_identity_10000() {
+    let test: Vec<u32> = (0..10000).collect();
+    assert_eq!(test, vlq::from_bytes(vlq::to_bytes(&test).as_slice()));
+}
+
+#[test]
+#[ignore]
+fn chained_execution_is_identity_lfsr() {
```

Comment:
Linear feedback shift register. I use it here to create pseudo random numbers. With the correct XOR value you can create a sequence of all numbers of size x-bits which wraps around.